soda/core/soda/execution/profile_columns_run.py

```diff
@@ -224,8 +249,12 @@ def profile_text_column(
             )
             text_aggregates_query.execute()
             if text_aggregates_query.rows:
-                profile_columns_result_column.distinct_values = int(text_aggregates_query.rows[0][0])
-                profile_columns_result_column.missing_values = int(text_aggregates_query.rows[0][1])
+                profile_columns_result_column.distinct_values = (
```

Comment:
same comment for here instead of float maybe we can use cast_int... helper

Reply:
yeah I'll make an even more generic one!